design/resources.go

```diff
@@ -535,7 +535,7 @@ var _ = a.Resource("work-item-link-type", func() {
 		a.Routing(
 			a.GET("/:id"),
 		)
-		a.Description("Retrieve work item link type (as JSONAPI) for the given ID.")
+		a.Description("Retrieve work item link type (as JSONAPI) for the given link ID.")
```

Comment:
@tsmaeder good catch. It is supposed to be "link type id". Fixed in local branch.